### ISSUE TITLE
from today to tomorrow afternoon

### ISSUE THREAD

**Issue by kkpoon** [wanasit/chrono#185]
in `from ... to ...` form

when one of the datetime is time aware and the other is only date. Its result is not correct

in this example, the `today` is not time aware and its hour follow `tomorrow` value
```
> chrono.parse("annual leave from today to tomorrow afternoon")[0]
ParsedResult {
  ref: 2017-04-06T15:42:00.868Z,
  index: 18,
  text: 'today to tomorrow afternoon',
  tags:
   { ENCasualDateParser: true,
     ENCasualTimeParser: true,
     ENMergeDateAndTimeRefiner: true,
     ENMergeDateRangeRefiner: true },
  start:
   ParsedComponents {
     knownValues: { day: 6, month: 4, year: 2017, hour: 15, minute: 0, second: 0 },
     impliedValues: { millisecond: 0 } },
  end:
   ParsedComponents {
     knownValues: { day: 7, month: 4, year: 2017, hour: 15, minute: 0, second: 0 },
     impliedValues: { millisecond: 0 } } }
```

in this example, `tomorrow` is not time aware and it doesn't make the `end` in result
```
> chrono.parse("annual leave from today morning to tomorrow")[0]
ParsedResult {
  ref: 2017-04-06T15:42:57.787Z,
  index: 18,
  text: 'today morning',
  tags:
   { ENCasualDateParser: true,
     ENCasualTimeParser: true,
     ENMergeDateAndTimeRefiner: true },
  start:
   ParsedComponents {
     knownValues: { day: 6, month: 4, year: 2017, hour: 6, minute: 0, second: 0 },
     impliedValues: { millisecond: 0 } } }
```

wish to able to identify the start and end has time component.

**Comment by wanasit** [comment]
Fixed in d6193a3.

**Comment by kkpoon** [comment]
Thanks👍🏿
